airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -773,12 +777,14 @@ def _clean(self, event: dict[str, Any]):
     def execute_complete(self, context: Context, event: dict, **kwargs):
         self.trigger_reentry(context=context, event=event)
 
-    def write_logs(self, pod: k8s.V1Pod):
+    def write_logs(self, pod: k8s.V1Pod, follow: bool = False, since_time: DateTime | None = None):
         try:
+            since_seconds = math.ceil((pendulum.now() - since_time).total_seconds()) if since_time else None
```

Comment:
We are trying to avoid using of `pendulum`

Reply:
right, I have removed pendulum. thanks